webpack.config.js

```diff
@@ -1,11 +1,7 @@
 var path = require('path');
 var webpack = require('webpack');
-var DedupePlugin = webpack.optimize.DedupePlugin;
-var UglifyJsPlugin = webpack.optimize.UglifyJsPlugin;
 var pkg = require('./package.json');
 
-var isDev = process.env.COINS_ENV === 'development';
```

Comment:
If we don't build in production then we might as well always minify.

Reply:
Sounds reasonable.